azure-pipelines/build.yml

```diff
@@ -0,0 +1,139 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+# This pipeline will be extended to the MicroBuild template
+# The Task 'NuGetCommand@2' has been converted to an output named 'NuGet push to VS feed' in the templateContext section.
+# The Task 'NuGetCommand@2' has been converted to an output named 'NuGet push to NuGet.org' in the templateContext section.
+name: $(Date:yyyyMMdd).$(Rev:r)
+variables:
+- name: BuildParameters.vsVersion
+  value: 17.0
+- name: Build.OfficialRelease
+  value: false
+- name: Codeql.Enabled
+  value: true
+- name: SignType
+  value: real
+- name: TeamName
+  value: vscx-tools-authoring
+- name: TimestampPackage
+  value: true
+trigger:
+  branches:
+    include:
+    - refs/heads/master
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    pool:
+      name: VSEngSS-MicroBuild2022-1ES
+    sdl:
+      sourceAnalysisPool:
+        name: AzurePipelines-EO
+        image: AzurePipelinesWindows2022compliantGPT
+        os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: stage
+      jobs:
+      - job: Phase_1
+        displayName: Agent job 1
+        cancelTimeoutInMinutes: 1
+        pool:
+          name: VSEngSS-MicroBuild2022-1ES
+        templateContext:
+          mb:
+            signing:
+              enabled: true
+              signType: real
+            sbom:
+              enabled: true
+          outputs:
+          - output: nuget
+            displayName: 'NuGet push to VS feed'
+            condition: and(succeeded(), eq(variables['Build.OfficialRelease'], 'true'))
+            packageParentPath: '$(Build.ArtifactStagingDirectory)'
+            searchPatternPush: src/Microsoft.Xaml.Behaviors/bin/Release/Microsoft.Xaml.Behaviors.Wpf.*.nupkg;!src/Microsoft.Xaml.Behaviors/bin/Release/Microsoft.Xaml.Behaviors.Wpf.*.symbols.nupkg
```

Comment:
I had numerous pipeline failures in `NuGet push` until I made following changes
•	Replace `searchPatternPush` with `packagesToPush`
•	Replace `feedPublish` with `publishVstsFeed`
•	Narrow packageParentPath to `$(...)\package` subfolder to avoid picking D:\a\_work\1\a\MicroBuild\Verify\in\package.